packages/xo-web/src/xo-app/home/backed-up-vms.js

```diff
@@ -0,0 +1,248 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Link from 'link'
+import PropTypes from 'prop-types'
+import React from 'react'
+import renderXoItem, { Vm } from 'render-xo-item'
+import SortedTable from 'sorted-table'
+import { addSubscriptions, connectStore, createCompare } from 'utils'
+import { compact, every, find, isEmpty, map, omit, some, uniq } from 'lodash'
+import { createGetObjectsOfType } from 'selectors'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { Text, XoSelect } from 'editable'
+import {
+  copyVms,
+  deleteVms,
+  editVm,
+  migrateVm,
+  migrateVms,
+  pauseVms,
+  restartVms,
+  snapshotVms,
+  startVms,
+  stopVms,
+  subscribeBackupNgJobs,
+  suspendVms,
+} from 'xo'
+
+const createCompareContainers = poolId =>
+  createCompare([c => c.$pool === poolId, c => c.type === 'pool'])
+
+const COLUMNS = [
+  {
+    name: _('name'),
+    itemRenderer: _ => <Vm id={_.id} link newTab />,
```

Comment:
Maybe we shouldn't use `Vm` in such a case since the whole row represents the VM, no just this column. That way, we could make the `name_label` also editable and the whole row would link to the VM. What do you think?